lib/internal/modules/esm/package_config.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const {
+  JSONParse,
+  SafeMap,
+  StringPrototypeEndsWith,
+} = primordials;
+const { URL, fileURLToPath } = require('internal/url');
+const {
+  ERR_INVALID_PACKAGE_CONFIG,
+} = require('internal/errors').codes;
+
+const packageJsonReader = require('internal/modules/package_json_reader');
+
+
```

Comment:
```suggestion

```
Extra line.

Reply:
counter nit 😅 i like the double line between deps and implementation. it helps visually break up the two distinct sections.

Reply:
I suppose it's also in other places so this can stay. :smile: